src/platform/src/main/java/org/geoserver/platform/resource/Paths.java

```diff
@@ -102,7 +102,41 @@ public static String path(String... path) {
         }
         return toPath(names);
     }
-
+    
+    // runtime flag which, if true, throws an error for the WARN characters
+    static final boolean STRICT_PATH = 
+            Boolean.valueOf(System.getProperty("STRICT_PATH", "true"));
+    
+    /**
+     * Pattern used to check for invalid file characters.
+     * <ul>
+     * <li> backslash
+     * </ul>
+     */
+    static final Pattern VALID = Pattern.compile("^[^\\\\]*$");
```

Comment:
Backslash is invalid on Windows. Forward slash should be invalid on Linux? http://stackoverflow.com/questions/9847288/is-it-possible-to-use-in-a-filename